influxdb_iox/tests/query_tests2/cases.rs

```diff
@@ -1,7 +1,41 @@
-use super::framework::{ChunkStage, TestCase};
+//! This module contains data driven tests that run sql queries from input files based .
+//!
+//! # Cookbook: Adding a new Test
+//!
+//! How to make a new test:
+//!
+//! 1. Add a new file .sql to the `cases/in` directory
+//! 2. Run the tests (e.g. `TEST_INTEGRATION=1 TEST_INFLUXDB_IOX_CATALOG_DSN=postgresql://postgres@localhost:5432/postgres cargo test --test end_to_end`)
+//! 3. You will get a failure message that contains commands of how to update the expected output file
+//!
+//! ## Example output
+//!
+//! ```
+//! Possibly helpful commands:
+//!   # See diff
+//!   diff -du "cases/in/pushdown.expected" "cases/out/pushdown.out"
+//!   # Update expected
+//!   cp -f "cases/in/pushdown.out" "cases/out/pushdown.expected"
+//! ```
+//!
+//! # Cookbook: Adding a new test scenario
+//!
+//! Each test can be defined in terms of a "setup" (a set of actions
+//! taken to prepare the state of database) which are defined in setups.rs.
+//!
+//! In the future, we envision more fine grained control of these
+//! setups (by implementing some of the database commands as IOX_TEST
+//! commands), but for now they are hard coded.
+//!
+//! The SQL files refer to the setups with a specially formatted comment:
+//!
+//! ```sql
+//! -- IOX_SETUP: OneMeasurementFourChunksWithDuplicates
+//! ```
+//!
+//! To add a new setup, follow the pattern in scenario.rs of `get_all_setups`.
```

Comment:
This is out of date and should say something like:

```suggestion
//! To add a new setup, follow the pattern in `influxdb_iox/tests/query_tests2/setups.rs`.
```